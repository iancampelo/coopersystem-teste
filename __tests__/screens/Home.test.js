import 'react-native'
import React from 'react'
import HomeScreen from '../../src/screens/Home'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'
jest.useFakeTimers()

describe('HomeScreen', () => {
  it('renders correctly HomeScreen', () => {
    const tree = renderer.create(<HomeScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })
});