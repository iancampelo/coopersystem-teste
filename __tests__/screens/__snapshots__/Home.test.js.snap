// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen renders correctly HomeScreen 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "marginTop": 50,
      }
    }
  >
    <Text>
      Recebendo lista de investimentos...
    </Text>
    <ActivityIndicator
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="large"
    />
  </View>
</View>
`;
